### ISSUE TITLE
UI gets slow with 4k

### ISSUE THREAD

**Issue by nexusV2** [koying/SPMC#760]
Hello Koying,

yesterday I switched spmc from 1080p (that felt pretty snappy) to 4k on a 4k Sony/Android TV and I wonder why (only) the UI gets slow, although even all the graphic elements stay at 1080p.

In other 4k Apps on my TV (Netflix/Amazon/Youtube) I didn´t recognised any UI or playback speed issues.
Is there any way to handle it?

Thank you for your great work, that I´m enjoying every day!!

**Comment by koying** [comment]
If you mean setting the GUI to "unlimited", there is no point on Sony.
It only makes the GUI slower (because SPMC renders in 4K) but the TV still downscales it to 1080p.

**Comment by nexusV2** [comment]
No, it´s a native 4k TV (Sony KD55 X8505) so it upscales everything by default to 4k. But when I switch from 1080p to 4k in SPMC (so that the TV doesn´t have to upscale from 1080p to 4k anymore) the UI is getting a bit sluggish and I don´t see what causes the additional heavy duty. As I mentioned above, the other 4k Apps I use are all doing fine.

**Comment by classicjazz** [comment]
The Sony X8505 reportedly has an ARMv7-based digital TV SoC from Mediatek with Mali-T624 graphics. The Mali-T624 chipset came out around 2012. It supports 4K **video** rendering but it isn't as capable as, say the Shield TV's Tegra X1 for **graphics** rendering. I would follow the Wiki's recommendations here for non-Shield Android devices:
https://github.com/koying/SPMC/wiki/Recommended-video-settings-for-Android-devices-%28other-than-Nvidia-Shield%29

**Comment by nexusV2** [comment]
Thank you. Now, after I read the Wiki carefully, this was exactly what I´m looking for. 
You have a great project and support!
